docs/content/using-npm/config.md

```diff
@@ -1311,6 +1313,34 @@ The program to use to view help content.
 
 Set to `"browser"` to view html help content in the default web browser.
 
+#### `which`
```

Comment:
`which` was not defined yet, and I re-ran the make to generate this file. We should think about automating this make command

Reply:
It's included in the `make docs` that's part of the release process as a final safety net, but some easier way to run it ahead of time would be nice, too, I agree.